src/Core/src/Handlers/Layout/LayoutHandler.Windows.cs

```diff
@@ -95,22 +104,33 @@ protected override LayoutPanel CreatePlatformView()
 			return view;
 		}
 
+		protected override void ConnectHandler(LayoutPanel platformView)
+		{
+			base.ConnectHandler(platformView);
+			_platformViewChildren = platformView.Children;
```

Comment:
So, the entire benefit here is this avoids calling the `FrameworkElement.Children` property over and over?

Do we have an idea what the property does underneath? Can we decompile it? Are they creating a copy of a native list -> managed on purpose?

I wonder if the native list will *move* and the cached one is out of date later.

/cc @PureWeen

Reply:
> "Do we have an idea what the property does underneath?"

Assuming that's some WinUI `Panel` control, accessing that property will invoke the `get_Children` method on the native WinRT object, then marshal the returned `IInspectable*` pointer to a C# object (ie. it'll go find the existing RCW for it, or create a new one if it doesn't exist already).

Reply:
@Sergio0694 any idea why this has such a big perf difference?

Reply:
It's just that in one case you're doing a bunch of work (two GC transitions, indirect call to the native method, the actual native method doing stuff, then all the marshalling work to get the RCW back). Technically speaking 2 more GC transitions and one more call to native code to also call `Release` on the raw native object. And possibly some more stuff. In the other you're just directly reading a field (as in, pretty much just a `mov`) 🙂

Reply:
Sounds like a bug! Slow code is no go code! Is there an issue currently tracking performance on the interop?

Reply:
I mean it's not a bug, it's just the expected behavior 😅
Yeah all WinRT issues go in the CsWinRT repo, you can always post there if you have any specific problems and we can help!

Reply:
That is more about supporting some additional scenarios and fixing some warnings, but it doesn't impact performance at all. If your code is working today, that PR will basically not do much at all for you (which is not a bad thing, just expected).

Reply:
> Sounds like a bug! Slow code is no go code! Is there an issue currently tracking performance on the interop?

I don't know how this would be a bug. Doing interop vs no-interop that is simply reading a field is going to be faster.